crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1478,6 +1488,7 @@ impl<'db> Type<'db> {
             Type::Unknown => Type::Unknown,
             // TODO map this to a new `Type::TypeVar` variant
             Type::KnownInstance(KnownInstanceType::TypeVar(_)) => *self,
+            Type::TypeAlias(alias) => alias.value(db),
```

Comment:
Note: this is the one method where we will want to map the type alias to its value.